app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    enum :role, [ :user, :admin ]
+    validates :first_name, :last_name, :email, :user_name, :encrypted_password, :phone, :role, presence: true
```

Comment:
what is encrypted_password? Can't we name it simply password?

Reply:
I have renamed encrypted_password to password in another branch.
I was planning to use user_name to sign in.

Reply:
Can first_name, last_name contain digits? Will the first_name `Ab12cd' be valid in your implementation?

Reply:
@nal-wtag  added regex for validation ([A-Za-z] only)